sdk/communication/Azure.Communication.Chat/src/autorest.md

```diff
@@ -14,7 +14,7 @@ If any of the new objects needs to be overwritten, add the required changes to t
 
 ``` yaml
 input-file:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/838c5092f11e8ca26e262b1f1099d5c5cdfedc3f/specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-09-21-preview2/communicationserviceschat.json
+    -  https://int.chatgateway.trafficmanager.net/swagger/2020-11-01-preview3/swagger.json
```

Comment:
Seems that the link here is forgotten to be updated to point to the spec repo

Reply:
For this one, I am not sure how to upload the latest swagger there. @minnieliu do you know .. or who can I ask ?

Reply:
You can send a new PR in the spec repo, or rev if you already have a PR open. Then if you go the file and choose "raw file" it will give you the link.